app/component/search/autosuggest.cjsx

```diff
@@ -99,6 +107,7 @@ class Autosuggest extends React.Component
         onSuggestionSelected={@onSuggestionSelected}
         inputAttributes={
           placeholder: @props.placeholder
+          onBlur: @disableInput
           #onBlur: @onSubmit Uh, causes a bug, as it is called after bluring the input after a selection has been done
```

Comment:
Not related to this commit, but what does that "causes a bug" comment mean? should we remove that since we now do not use it?